### PR TITLE
ARIA form role: exposed if it has an accessible name

### PR DESCRIPTION
**Pull request by estelle**
https://www.w3.org/TR/html-aria/#el-form

### CODE REVIEW

File: files/en-us/web/html/element/form/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ browser-compat: html.elements.form
 
 {{HTMLSidebar}}
 
-The **`<form>`** [HTML](/en-US/docs/Web/HTML) element represents a document section containing interactive controls for submitting information.
+The **`<form>`** [HTML](/en-US/docs/Web/HTML) element represents a document section containing interactive controls for submitting information. The form is not exposed as a landmark region unless it has an [accessible name](/en-US/docs/Glossary/Accessible_name).
```
Comment (chrisdavidmills) [inline files/en-us/web/html/element/form/index.md:10]: This change looks correct to me, but I can't help wondering whether the average reader will have a clue what a landmark region is. Would it be even better to link to something that explains what it is and how to give it an accessible name, and/or provide more clues in the text, for example, something like

>  The form is not exposed to [screen readers](/en-US/docs/Glossary/Screen_reader) as a landmark region unless it has an [accessible name](/en-US/docs/Glossary/Accessible_name) provided via the `name` attribute.

?

Also, does a form always need a specific accessible name, or is this only needed in some circumstances, e.g. when you have multiple forms on a page? If there is only one form and the top-level heading for the pages clearly says this page contains a form, isn't it OK to not include this as long as the individual form elements are clearly labelled? 

 I feel like I'm being horribly picky now, but it seems to me like this statement needs to come with some advice on when this is necessary. And, if it is necessary to always include this, examples on the page and elsewhere need to be updated to align with this advice.

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
@estelle hrm, I feel like I've made a mountain out of a molehill with this one. Let me know what you think!

**Comment by estelle** [comment]
Thanks for the review. This PR was a mistake. I meant to add the sentence to the form role not form element. 
closed this and opened:
https://github.com/mdn/content/pull/29665

**Comment by chrisdavidmills** [comment]
> Thanks for the review. This PR was a mistake. I meant to add the sentence to the form role not form element. closed this and opened: #29665

Ah, that makes more sense ;-)

**Pull request closed without merging**